manifest.json

```
@@ -0,0 +1,8 @@
+{
+    "domain": "dreamscreen",
+    "name": "DreamScreen",
+    "documentation": "https://github.com/GregoryDosh/Home-Assistant-DreamScreen-Service",
+    "dependencies": [],
+    "codeowners": ["https://github.com/GregoryDosh"],
+    "requirements": ["PyDreamScreen.HA>=0.0.9"]
```

This can be changed back once the upstream PR gets approved:

```suggestion
    "requirements": ["PyDreamScreen>=0.0.9"]
```